### ISSUE TITLE
Invoke remote method timeout调用超时原因？

### ISSUE THREAD

**Issue by duguyixiaono1** [alibaba/dubbo#1093]
监控上观察到，某些个请求的调用链，从自定义的filter（有埋点）到接口实现AOP（有埋点）之间耗时6s，导致服务调用超时，请问，有什么可能的原因？或是排查思路？

**Comment by chickenlj** [comment]
This is a problem that is closely related to your environment and project.
And besides, i don't fully understand the problem through your description, you mean all 6 seconds cost on the provider side?

**Comment by duguyixiaono1** [comment]
@chickenlj 
yes, all 6 seconds cost on the provider side between custom filter and implement.
It is not a normal invocation chain, but it was really captured by our monitor service. 
Is there any possible time-consuming process between them.
